SR2019/2019-05-15-kit-chase-round-1.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Teams with outstanding kit
+subject: Outstanding kit
+---
+
+Hello!
+
+According to our records, you still have a kit from this years competition. As the competition has ended, we would like this back!
+
+Please securely pack the kit and ship it to the address listed on [our website](https://studentrobotics.org/contact/). Once you have dispatched it, please [let us know](mailto:teams@studentrobotics.org).
+
+## How to ship your kit
```

Comment:
Is SR willing to reimburse/cover the cost of this? If not I think it's ok to omit that, but if we are willing we should probably say so and suggest how we'll do so.

Reply:
SR will *not* be covering the cost